tests/testDipoleFitter.py

```diff
@@ -56,86 +55,74 @@ class DipoleFitTestGlobalParams(object):
     @var offsets: pixel coordinates between lobes of dipoles
     """
 
-    def __init__(self):
+    def __init__(self, xc=np.array([65.3, 24.2]), yc=np.array([38.6, 78.5])):
         """Initialize the parameters."""
         np.random.seed(666)
         self.display = False
         self.verbose = False
         self.w, self.h = 100, 100  # size of image
 
-        self.xc = np.array([65.3, 24.2])  # xcenters of two dipoles in image
-        self.yc = np.array([38.6, 78.5])  # ycenters of two dipoles
+        self.xc = xc  # xcenters of two dipoles in image
+        self.yc = yc  # ycenters of two dipoles
         self.flux = np.array([2500., 2345.])  # fluxes of pos/neg lobes
         self.gradientParams = np.array([10., 3., 5.])
 
         self.offsets = np.array([-2., 2.])  # pixel coord offsets between lobes of dipoles
+        self.testImage = ipUtils.DipoleTestImage(
+            xcenPos=self.xc + self.offsets,
+            ycenPos=self.yc + self.offsets,
+            xcenNeg=self.xc - self.offsets,
+            ycenNeg=self.yc - self.offsets,
+            flux=self.flux, fluxNeg=self.flux,
+            gradientParams=self.gradientParams)
+
+        self.catalog = self.testImage.detectDipoleSources()
 
 
 # First, test the algorithm itself (fitDipole()):
```

Comment:
Again: comments like this should be part of the docstring for the class. This makes them discoverable.